File: codeforces/38.anton_and_letters/aal.py
# Author: José Rodolfo (jric2002)
letters = str(input())
letters = letters[1:(len(letters) - 1)]
letters = letters.split(", ")
different_letters = []
if (letters[0] == ""):
  letters = []
for letter in letters:
  if letter not in different_letters:
    different_letters.append(letter)
print(len(different_letters))

File: codeforces/38.anton_and_letters_[443A]/aal.cpp
#include <iostream>
#include <cstring>
/* Author: José Rodolfo (jric2002) */
using namespace std;
int main() {
  string letters, different_letters = "";
  char letter;
  bool exist;
  unsigned short int size_letters, letter_position;
  getline(cin, letters);
  size_letters = letters.size();
  if (size_letters > 2) {
    for (unsigned short int i = 1; i < size_letters; i += 3) {
      letter = letters[i];
      letter_position = 0;
      exist = false;
      while (letter_position < different_letters.size()) {
        if (letter == different_letters[letter_position]) {
          exist = true;
          break;
        }
        letter_position += 1;
      }
      if (!exist) {
        different_letters += letter;
      }
    }
  }
  cout << different_letters.size() << endl;
  return 0;
}